content/components/page-header.mdx

```diff
@@ -299,6 +299,74 @@ The title bar can include a static page title, actions, and leading and trailing
+It’s important to note that if you utilize a component with an existing `navigation` landmark such as `UnderlineNav` then you should not use `PageHeader.Navigation` as a navigation landmark, as this would be a duplicate.
+
+For navigation, you should follow these guidelines: 
+- If you have a list or grouping of links within `PageHeader.Navigation`, you should utilize the `nav` type for this element.
+- If you’re already utilizing a component with a navigation landmark (such as `UnderlineNav`), you should not use `PageHeader.Navigation` as a navigation landmark.
+- If more than one `navigation` landmark exists on the page at a given time, you should add a unique label to each one.
+You can do this by either passing an `aria-label` or `aria-labelledby` attribute to `PageHeader.Navigation`.
+
+### Heading Structure
+
+`PageHeader.Title` renders its content as a heading in the background and most of the time it will have crucial information about the page, so it’s important to ensure that you’re using it in an accessible manner.
+
+`PageHeader.Title` is shipped as an `H2` by default, but only if you do not pass a heading level to it via the `as` prop. You should modify the level of this component if the default heading `H2` does not make sense with the existing heading structure on the page.
+
+### Dom Ordering
+
+When PageHeader has a context area and/or a leading action, the dom ordering of the component needs to be carefully considered to ensure that the page header is accessible to all users.
+To be able to achieve this, we need to make sure that all of the interactive content in the page header comes after the main heading of the page (PageHeader.Title).
+With this approach, we can ensure that the screen reader users who are navigating through the heading menu don't miss any of those interactive content.
+
+Note that the visual and DOM ordering of the elements can only differ if the page heading never contains one or more focusable elements. If the heading includes any focusable elements, the visual and the DOM order must match.
+
+The recommended dom order of the page header components is:
+
+```jsx
+<PageHeader.TitleArea>
+  <PageHeader.LeadingVisual>...</PageHeader.LeadingVisual>
+  <PageHeader.Title as="h1">...</PageHeader.Title>
+  <PageHeader.TrailingVisual>...</PageHeader.TrailingVisual>
+</PageHeader.TitleArea>
+<PageHeader.ContextArea>
+  <PageHeader.ParentLink>...</PageHeader.ParentLink>
+  <PageHeader.ContextBar>...</PageHeader.ContextBar>
+  <PageHeader.ContextAreaActions>...</PageHeader.ContextAreaActions>
+</PageHeader.ContextArea>
+<PageHeader.LeadingAction>...</PageHeader.LeadingAction>
+<PageHeader.TrailingAction>...</PageHeader.TrailingAction>
+<PageHeader.Actions>...</PageHeader.Actions>
+<PageHeader.Description>...</PageHeader.Description>
+<PageHeader.Navigation>...</PageHeader.Navigation>
+```
+
+<img
+  src="https://github.com/primer/design/assets/1446503/3cf81858-f76c-4bfd-af4f-f800a82fd135"
```

Comment:
I made this in Figma but I am not 100% sold in this. The image is too big and not sure if it communicated the dom ordering well. I would love any feedback on this. We can remove it if it is not helpful too.

Reply:
I think we can just remove it.